plugins/parsers/json/README.md

```diff
@@ -30,10 +30,12 @@ ignored unless specified in the `tag_key` or `json_string_fields` options.
   json_query = ""
 
   ## Tag keys is an array of keys that should be added as tags.  Matching keys
-  ## are no longer saved as fields.
+  ## are no longer saved as fields. Now supports wildcard glob matching.
```

Comment:
Maybe leave out the "Now" as this might be outdated in a few month.

Reply:
Done